test/nuts/mdapi.nut.ts

```diff
@@ -58,6 +74,58 @@ describe('mdapi NUTs', () => {
     });
   });
 
+  describe('mdapi:beta:convert', () => {
```

Comment:
Now that we have both convert commands in here, I'd be more interested in a round-trip conversion NUT on some decently-complex repo, with an assertion that everything matched.

Since there's no org involved, it'd run pretty fast.

Reply:
Yeah, we could do that.  I know the ebikes-lwc source is not in the expected format in the repo (due to 1 space in the xml header) but maybe dreamhouse is.  Regardless, we could source:convert, mdapi:convert, then source:convert again and compare.